code/controllers/subsystem/persistence.dm

```diff
@@ -188,27 +194,73 @@ SUBSYSTEM_DEF(persistence)
 
 	return json
 
-
-/datum/controller/subsystem/persistence/proc/LoadTrophies()
+/// Loads the trophies from the source file, and places a few in trophy display cases.
+/datum/controller/subsystem/persistence/proc/load_trophies()
+	var/list/raw_saved_trophies = list()
 	if(fexists("data/npc_saves/TrophyItems.sav")) //legacy compatability to convert old format to new
```

Comment:
Could probably get rid of this at this point, it was a while since.

Reply:
Done, it has probably been converted everywhere by now.